html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form id="sign-up-form" action="" method="get">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" id="email" name="email-address" />
+      <div>
+        <label for="why-coding">Why is coding cool?</label>
+      </div>
+      <select name="why-coding-is-cool">
+        <option id="select-option" value="select" selected="selected">
+          <label for="select-option">Select an option</label>
```

Comment:
you do not need labels for the individual options in a select, just the select itself

Reply:
Oh! Will fix! 
Is that true for the radio buttons as well? The graphic on pg163 of the Duckett book may have led me astray. 
![image](https://user-images.githubusercontent.com/85596835/144112448-350c9226-6aa2-4433-820f-f7bb4414e98a.png)